### DISCUSSION TITLE
Menu entry swap compass to custom camera directions when you click it

### DISCUSSION THREAD

**Discussion by EVIGMA** [runelite/runelite#15870] [Ideas]
Can you customise the compass so when we left click, right click and shift click it, it turns the camera a specified direction (North, South, East or West). 

It will make activities like lava rc more enjoyable when pointing east is beneficial while making the runes.

**Comment by SaffiCauthon** [comment]
Best to use [Camera Points plugin](https://github.com/NullNameException/camera-points) in the plugin-hub for RC.
![Camera Points Example](https://i.imgur.com/otfTWqO.png)

Similarly, you can use [Custom Menu Swaps](https://github.com/geheur/More-menu-entry-swaps).
On a new line in your custom swap, type the UI option like `Look east`.
![Custom Swap Example](https://i.imgur.com/31fuoTo.png)
